features/rails_features/integrations.feature

```diff
@@ -71,10 +71,10 @@ Scenario: Rake
 
 @rails_integrations
 Scenario: Resque (no on_exit hooks)
-  When I run "bundle exec rake resque:work" in the rails app
-  And I run "Resque.enqueue(ResqueWorker, 123, %(abc), x: true, y: false)" with the rails runner
-  And I wait to receive a request
-  Then the request is valid for the error reporting API version "4.0" for the "Ruby Bugsnag Notifier"
+  When I run "Resque.enqueue(ResqueWorker, 123, %(abc), x: true, y: false)" with the rails runner
+  And I run "timeout --signal QUIT 5 bundle exec rake resque:work" in the rails app
```

Comment:
[`timeout` uses the `TERM` signal by default](https://www.man7.org/linux/man-pages/man1/timeout.1.html), which causes Resque to report an error (and the test to fail as it gets 2 events)

[Resque gracefully shutsdown when given the `QUIT` signal](https://github.com/resque/resque#signals), so we have to tell `timeout` to do that instead